src/index.ts

```diff
@@ -1,7 +1,114 @@
+import sum from 'ml-array-sum';
+import variance from 'ml-array-variance';
+import covariance from 'ml-array-xy-covariance';
+import { Matrix, EigenvalueDecomposition } from 'ml-matrix';
+
 /**
- * My module
- * @returns A very important number
+ * Calculates the Directional Ellipse for a set of points with a specific standard deviation
+ * @param {object} [points]
+ * @param {Array} [points.x]
+ * @param {Array} [points.y]
+ * @param {number} [nbSD=2]
+ * @param {object} [props={fillColor='#FF0',fillOpacity=0.5,strokeWidth=0.00001,strokeColor='#F00',strokeDasharray=[1]}]
  */
-export function myModule(): number {
-  return 42;
+export function getDirectionalEllipse(
+  points: { x: number[]; y: number[] },
```

Comment:
we like to accept an array of number of a float64 typed array. You can use as type https://github.com/cheminfo/cheminfo-types/blob/main/src/index.d.ts#L4 (and the project can depend on cheminfo-types)